benchmark/opperf/rules/default_params.py

```diff
@@ -58,6 +58,11 @@
 DEFAULT_LAM = [[1.0, 8.5]]
 DEFAULT_K_ND = [[20, 49]]
 DEFAULT_P_ND = [[0.4, 0.77]]
+DEFAULT_GRID = [(32, 2, 256, 256)]
+DEFAULT_DATA_BILINEAR = [(32, 2, 256, 256)]
+DEFAULT_TRANSFORM_TYPE = ['warp']
```

Comment:
what if we add other transform types too?

Reply:
Great point, adding now.